### PR TITLE
Fix capacity nameplate for Battery OM costs in Fuel Cell

### LINKED ISSUE #520: Fuel Cell battery OM costs using wrong capacity value
**Describe the bug**
The nameplate capacity used for the battery OM Costs in the PV-Fuel Cell-Battery config appears to be using the kWh capacity rather than the kW capacity. This is not the correct usage as used in the financial models. 

Expected behavior: The nameplate variable om_capacity1_nameplate should be calculated from batt_power_discharge_max_kwdc or batt_power_discharge_max_kwac as is done in the capital costs (battery_power). 

image

image

image

**Additional context**

### PR DESCRIPTION
**Pull request by mjprilliman**
-Changes capacity nameplate definition from kWh to kW for battery OM cost calculations
-Closes #520

### CODE REVIEW

**Review by cpaulgilman — CHANGES_REQUESTED** [review]
Thanks for finding this issue. The "Fuel Cell O and M Costs" UI form is only used for Fuel Cell configurations, so I think the checks for PV Battery and PVWatts Battery configurations can be omitted.

The O&M cost inputs for fuel cell are still a bit confusing because the replacement cost in $/kWh appears under the kW capacity column, but we can fix that later when we clean up the O&M variable names for https://github.com/NREL/SAM/issues/355.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (sjanzou) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Why are these files here?

File: deploy/runtime/ui/Fuel Cell O and M Costs.txt [hunk 2]
```diff
@@ -2469,7 +2469,16 @@ equations{ 'add_om_num_types'} = define(){
 	return 2;
 };
 equations{ 'om_capacity1_nameplate'} = define(){
-	return ${batt_computed_bank_capacity};
+	if( technology() == 'PV Battery' || technology() == 'Fuel Cell')
+			return ?${en_batt}
```
Comment (sjanzou) [inline deploy/runtime/ui/Fuel Cell O and M Costs.txt:2473]: what about generic battery?

**Review by sjanzou — CHANGES_REQUESTED** [review]
Why are the but_report and request_feature files to be merged into patch?
Why are the epw and csv files to be merged into patch - are they part of new defaults?
Was the Sandia module library updated in this pull request?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mjprilliman) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: I accidentally branched off of develop originally then tried to change the branch to patch here. I may need to redo this. I'm not sure why bug_report and request_feature are here.

File: deploy/runtime/ui/Fuel Cell O and M Costs.txt (hunk 2 above)
Comment (cpaulgilman) [reply deploy/runtime/ui/Fuel Cell O and M Costs.txt:2473]: @sjanzou This is on the Fuel Cell O and M Costs form that is only used for the Fuel Cell configurations, so no need for the configuration checks for other battery techs here.
